### PR TITLE
Fix #10 - Removed coffeescript

### LINKED ISSUE #10: Changes on DB not affect views
I'm listing a collection on view but if I open a mongodb gui (like robomongo or mongohub) and do a update, in the view the field updated turn in a blank field, my code:

`lib/Players.js`
```javascript
Players = new Mongo.Collection("players");
```

client/players.js
```javascript
Template.players.rendered = function(){
    var vm = new Vue({
        el: '#vue',
        data : {
            players : []
        },
        sync : {
            players : function(){
                return Players.find().fetch();
            }
        },
        methods : {
            update : function(doc){
                Players.update({_id:doc._id}, {$set:{
                    nome : doc.nome + ' - UPDATED'
                }});
            }
        }
    });
}
```

`client/players.html`
```html
<template name="players">
    <table class="table table-stripped" id="vue">
        <tbody>
            <tr v-repeat="doc in players">
                <td>[[doc._id]]</td>
                <td>[[doc.nome]]</td>
            </tr>
        </tbody>
    </table>
</template>
```

Before update:

Update:

After update:

Packages on that sample project:
```
meteor-platform
autopublish
less
iron:router
twbs:bootstrap@3.3.5
fortawesome:fontawesome
zimme:iron-router-active
jquery
okgrow:iron-router-autoscroll
vue:vue
```

Using the method `update` works fine.

### PR DESCRIPTION
**Pull request by nerijunior**
VUE updated to version 1.0
And the issue #10 was fixed!

### CODE REVIEW

File: lib/main.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    if (typeof rxFunc === 'function') {
+        var self = this;
+
+        this.$$syncDict[key] = Tracker.autorun(function(){
+            var val;
+
+            val = rxFunc.call(self);
+
+            if (val) {
+                if (val.fetch != null) {
```
Comment (zhouzhuojie) [inline lib/main.js]: `if (val.fetch != null && typeof val.fetch === 'function')

**Comment by zhouzhuojie** [comment]
lgtm! Thanks so much for upgrading this!

File: lib/main.js (hunk 1 above)
Comment (nerijunior) [inline lib/main.js]: Added!

**Comment by zhouzhuojie** [comment]
Just curious the root cause of #10 , why Vue@v1 will not cause the same issue

**Comment by nerijunior** [comment]
@zhouzhuojie With 0.12 or 1.0 the issues happens I only update by the opportunity.

**Comment by Codebryo** [comment]
Thx @nerijunior I hope the code I previously made was helpful. Awesome to see everything back up running :) Thx for this!

**Pull request merged**